pylint/lint/pylinter.py

```diff
@@ -622,8 +261,7 @@ def __init__(
         self.stats = LinterStats()
 
         # Attributes related to (command-line) options and their parsing
-        self._external_opts = options
```

Comment:
This was not used anywhere?

Reply:
No! I double-checked. It's private so we can remove it without any hassle, the first time I think 😄